src/Components/Inspirations/Inspirations.tsx

```diff
@@ -1,24 +1,56 @@
+import React, { useEffect, useState } from 'react';
 import './Inspirations.scss';
 import { InspirationImage } from './InspirationImage';
 import { InspirationText } from './InspirationText';
+import Timer from '../Timer/Timer'
+import ChooseTime from '../ChooseTime/ChooseTime'
```

Comment:
Moving the Timer and ChooseTime components inside of the Inspirations component makes this easier to style